tmt/templates/default/plan.j2

```diff
@@ -0,0 +1,5 @@
+{{ plan_name }}:
+    discover:
+        how: fmf
+    execute:
+        how: tmt
```

Comment:
So this is a `plan` template. What about moving it under `plan/default.j2` to be with other plan templates? Or is there a special reason why to keep it separately here?

Reply:
I thought this is a special case because the plan needs the `plan_name` variable to render correctly. If this plan would be inside the `plan` directory, it would be available as one of the default templates users can use. However, users can't set the variable using the command line, so they would have to edit the created plan manually.

Reply:
Ah, right, makes sense.